templates/create_initial_root_token.rb.epp

```diff
@@ -0,0 +1,21 @@
+<%-|
+  Optional[Sensitive[String[1]]] $token,
+  Integer[0] $token_ttl_minutes,
+  Stdlib::AbsolutePath $token_file_path,
+|-%>
+# This scripts creates an initial root token and stores it to the
```

Comment:
how about a shebang line?

Reply:
It's executed through the Gitlab Rails Runner, so it shouldn't need one.